src/data/faq_de.json

```diff
@@ -2220,8 +2220,8 @@
                                 "active": false,
                                 "textblock": [
                                     "<b>AKTUELL 14. Januar 2021</b><br/>Sollten Sie dieses Problem haben, schreiben Sie bitte Ihren Bericht hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/1962' target='_blank' rel='noopener'>cwa-app-android/issues/1962</a>.<hr/>",

```

Comment:
Should be rel='noopener noreferrer'

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3032/commits/2180bce04f3ca6fcfd3f0e9d26b27cae68ebebe1, thanks!